### PR TITLE
Adding non-empty and duplicate project name check to create project mutation

### PR DESCRIPTION
**Pull request by rajdas98**
## Proposed changes

Adding non-empty and duplicate project name check to create project mutation

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/graphql-server/pkg/project/create.go [hunk 1]
```diff
@@ -34,6 +34,21 @@ func CreateProjectWithUser(ctx context.Context, projectName string, userID strin
 		return nil, err
 	}
 
+	if projectName == "" {
+		return nil, errors.New("project name can't be empty")
+	}
+
+	//checking duplicate projectname
+	filter := bson.D{{"name", projectName}}
```
Comment (rajdas98) [inline litmus-portal/graphql-server/pkg/project/create.go:42]: Need to add userid also.

Comment (rajdas98) [reply litmus-portal/graphql-server/pkg/project/create.go:42]: adding it

**Comment by SarthakJain26** [comment]
@rajdas98 There's one more mutation to update project name `updateProjectName`. You will have to update the conditions there as well

File: litmus-portal/graphql-server/pkg/project/create.go (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/graphql-server/pkg/project/create.go:42]: @rajdas98 Do we require userid? The project name should be unique across all users.

Comment (rajdas98) [reply litmus-portal/graphql-server/pkg/project/create.go:42]: cc: @ajeshbaby

**Review by SarthakJain26 — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Pull request merged**